### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by kwsorensen**

### CODE REVIEW

**Comment by kwsorensen** [comment]
image
Screenshot of changes running the script locally

**Comment by laverya** [comment]
Could we get some tests of validate like this? https://github.com/replicatedhq/kURL/blob/b5800db1880d5a2a7f6f65a21a8e948f80fc7aaf/web/src/test/controllers/installers.ts#L783-L797

Making sure that "latest" rook + "latest" k8s fails

File: web/src/client/index_test.ts [hunk 1]
```diff
@@ -654,21 +662,25 @@ describe("GET /<installerID>", () => {
...
-
+    
     it(`injects k8s ${latestResolve.spec.kubernetes.version}, weave ${latestResolve.spec.weave?.version}, rook ${latestResolve.spec.rook?.version}, contour ${latestResolve.spec.contour?.version}, registry ${latestResolve.spec.registry?.version}, prometheus ${latestResolve.spec.prometheus?.version}`, async () => {
       const script = await client.getJoinScript("latest");
 
       expect(script).to.match(new RegExp(`version: ${latestResolve.spec.kubernetes.version}`));
       expect(script).to.match(new RegExp(`version: ${latestResolve.spec.weave?.version}`));
-      expect(script).to.match(new RegExp(`version: ${latestResolve.spec.rook?.version}`));
+      expect(script).to.match(new RegExp(`version: ${latestResolve.spec.minio?.version}`));
+      expect(script).to.match(new RegExp(`version: ${latestResolve.spec.longhorn?.version}`));
```
Comment (emosbaugh) [inline web/src/client/index_test.ts:675]: does this list need to include containerd?

**Review by emosbaugh — APPROVED** [review]

File: web/src/client/index_test.ts (hunk 1 above)
Comment (kwsorensen) [reply web/src/client/index_test.ts:675]: added it

**Pull request merged**